response_operations_ui/__init__.py

```diff
@@ -26,17 +31,31 @@
 app.secret_key = app.config['RESPONSE_OPERATIONS_UI_SECRET']
 
 logger_initial_config(service_name='response-operations-ui', log_level=app.config['LOGGING_LEVEL'])
+logger = logging.getLogger(__name__)
 
 login_manager = LoginManager(app)
 login_manager.init_app(app)
 login_manager.login_view = "sign_in_bp.sign_in"
 
+if app.config['SESSION_TYPE'] == 'redis':
+    # If deploying in cloudfoundry set config to use cf redis instance
+    if cf.detected:
+        logger.info('Cloudfoundry detected, setting service configurations')
+        app.config['REDIS_HOST'] = cf.redis.credentials['host']
+        app.config['REDIS_PORT'] = cf.redis.credentials['port']
+
+    redis = redis.StrictRedis(host=app.config['REDIS_HOST'],
+                              port=app.config['REDIS_PORT'],
+                              db=app.config['REDIS_DB'])
+
+    # wrap in the flask server side session manager and back it by redis
+    app.config['SESSION_REDIS'] = redis
+
+Session(app)
```

Comment:
By wrapping the app with Session all endpoints will create a session, including `/info` - if this endpoint is hit by an automated check e.g. to check if the service is still up, then we could end up with a lot of sessions being created.

Reply:
Good spot, will investigate remove certain URLs

Reply:
It's difficult to remove urls from the flask session as it's app based. However i've cleared the session if you hit the info endpoint, that should stop them building up unnecessarily in REDIS. The only downside being if the user manual hits /info it'll sign them out. Though they shouldn't be doing that anyway.

Reply:
I think this is fine for now.